andela/caret-bn-backend
# #168805154-add pre hooks

#### What does this PR do?
- Add pre-hooks to check errors and if the tests are passing before commit and push.
#### Description of Task to be completed
  
- I have installed husky to run pre-commit and pre-push hooks.
- I added hooks script in package.json.

#### How should this be manually tested?
- clone the repository : git clone https://github.com/andela/caret-bn-backend.git
- checkout to branch ch-add-pre-hooks-168805154
- do npm install
- try to commit you will see pre hooks running before commit


#### Any background context you want to provide?
    `N/A`
#### What are the relevant pivotal tracker stories?
[#168805154](https://www.pivotaltracker.com/n/projects/2400651)

#### Screenshots (if appropriate)
<img width="700" alt="Screen Shot 2019-10-02 at 00 47 25" src="https://user-images.githubusercontent.com/47029978/66006234-4388ec00-e4ae-11e9-879e-037efa408191.png">

####  Questions:
   `N/A`
